getting-started/templates/systemlink-secrets.yaml

```diff
@@ -123,7 +123,7 @@ userservices:
         - serviceName: "saltmaster-init"
           key: ""
           hash: "<SHA512 hash of key>"
-        - serviceName: "sessionmanagerservice"
+        - serviceName: "sessionmanager"
```

Comment:
Don't do this. This is the API key name which is not changing.

Reply:
So we use the serviceName here for the API key name?

Reply:
@stick152 It appears this hadn't been updated yes, and so doesn't need to change.

Reply:
Okay, so this was already done in regard to Chris' bug.  I think I should just abandon this change.